packages/web-runtime/src/components/Topbar/TopBar.vue

```diff
@@ -53,6 +55,17 @@ export default {
       default: () => []
     }
   },
+  setup() {
+    const notificationsSupport = useCapabilityNotifications()
+
+    const isNotificationBellEnabled = computed(() => {
+      return unref(notificationsSupport)?.includes('list') || false
```

Comment:
`includes()` be enough now, because it defaults to `[]`.

```suggestion
      return unref(notificationsSupport).includes('list')
```

Reply:
thought it would be safer that way but fine too